qt/DetailsDialog.cc

```diff
@@ -164,7 +165,7 @@ class PeerItem : public QTreeWidgetItem
                 if (ipAddress.protocol() == QAbstractSocket::IPv4Protocol)
                 {
                     quint32 const ipv4Address = ipAddress.toIPv4Address();
-                    collatedAddress = QLatin1String("1-") + QString::fromLatin1(QByteArray::number(ipv4Address, 16).
+                    collatedAddress = QLatin1String("1-") % QString::fromLatin1(QByteArray::number(ipv4Address, 16).
```

Comment:
This (and lots of other stuff) is not something related directly to editDate support, I'd rather split this PR in two if you want to make additional refactoring preceding or following the advertised changes.